.github/workflows/python-package.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Language Learn
+
+defaults:
+  run:
+    working-directory: api
+
+on:
+  push:
+  pull_request:
+    branches: [ "main", "dev", "release" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.9"]
+
```

Comment:
If python version is different then will it show an error or what?

Reply:
I think it shouldn't show an error and if it does then what error it gives?

Reply:
If you want to learn more about Github Actions they've got a decent introduction here:
https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions